Requirements/ViewModels/RequirementBrowser/Rows/RequirementsGroupRowViewModel.cs

```diff
@@ -74,16 +74,7 @@ public class RequirementsGroupRowViewModel : RequirementContainerRowViewModel<Re
         /// </summary>
         public string Categories
         {
-            get
-            {
-                if (this.Thing == null)
-                {
-                    return string.Empty;
-                }
-
-                var firstOrDefault = this.Thing.Category.FirstOrDefault();
-                return firstOrDefault != null ? string.Join(", ", this.Thing.Category.Select(x => x.Name)) : string.Empty;
-            }
+            get => this.Thing.Category.Any() ? string.Join(", ", this.Thing.Category.Select(x => x.Name)) : string.Empty;
```

Comment:
Well, not exactly, You will get a `NullReferenceException` if `Thing` is null. So either leave the null check or integrate it into the oneliner.